### PR TITLE
Hello_world

### PR DESCRIPTION
**Pull request by mitkotak**
Loading up hello_world job by default.

### CODE REVIEW

File: server.ts [hunk 1]
```diff
@@ -126,6 +126,36 @@ async function prepareDataForDB(data, properties) {
...
+    const helloWorldGit = await connection.getRepository(Git).findOne('hello_world')
+    
+    if (!helloWorldGit) {
+    
+    const git = {
+    
+    id: 'hello_world',
+    
+    address: `https://github.com/cybergis/cybergis-compute-hello-world.git`,
+
```
Comment (zimo-xiao) [inline server.ts]: The code looks great. Though, could you work on the indentations and remove some extra lines in the code? It seems not well formatted. Overall, code looks good to me.

File: server.ts [hunk 2]
```diff
@@ -126,6 +126,23 @@ async function prepareDataForDB(data, properties) {
...
+    if (helloWorldGit == undefined) {
+    const git = {
+    id: 'hello_world',
+    address: `https://github.com/cybergis/cybergis-compute-hello-world.git`,
+    isApproved: true,
+    createdAt: new Date(),
+    updatedAt: new Date()
+    }
+    await connection.getRepository(Git).save(git) 
+    }}
```
Comment (zimo-xiao) [inline server.ts]: I don't think this is well formatted. If you are using VS Code, you can just type Shift + Option + F to format the code.

Reference: https://stackoverflow.com/questions/29973357/how-do-you-format-code-in-visual-studio-code-vscode

Comment (mitkotak) [reply server.ts]: How does it look now ? @zimo-xiao It reformatted a large part of the codebase.

**Comment by zimo-xiao** [comment]
lgtm

**Pull request merged**